velox/docs/functions/bitwise.rst

```diff
@@ -17,3 +17,28 @@ Bitwise Functions
 .. function:: bitwise_xor(x, y) -> [bigint]``
 
     Returns the bitwise XOR of ``x`` and ``y`` in 2's complement representation.
+
+.. function:: bitwise_arithmetic_shift_right(x, shift) -> [bigint]``
```

Comment:
Functions in the docs should appear in alphabetical order. Would you make it so?